src/bootstrap/sanity.rs

```diff
@@ -120,10 +120,9 @@ pub fn check(build: &mut Build) {
     }
 
     build.config.python = build.config.python.take().map(|p| cmd_finder.must_have(p))
-        .or_else(|| cmd_finder.maybe_have("python2.7"))
-        .or_else(|| cmd_finder.maybe_have("python2"))
+        .or_else(|| cmd_finder.maybe_have("python3"))
```

Comment:
Same here, I'd prefer to avoid breaking compat with python 2, we should keep it as a fallback everywhere (in bootstrap/configure/etc, our CI scripts can assume it's installed).